src/systems/thruster/Thruster.hh

```diff
@@ -53,15 +53,18 @@ namespace systems
   ///   [Optional]
   /// - <joint_name> - This is the joint in the model which corresponds to the
   ///   propeller. [Required]
-  /// - <use_angvel_cmd> - Default false, if set to true will make the thruster
+  /// - <use_angvel_cmd> - If set to true will make the thruster
   ///   plugin accept commands in angular velocity in radians per seconds in
-  ///   terms of newtons. [Optional, Boolean, Default True]
+  ///   terms of newtons. [Optional, Boolean, defaults to false]
   /// - <fluid_density> - The fluid density of the liquid in which the thruster
   ///   is operating in. [Optional, kg/m^3, defaults to 1000 kg/m^3]
   /// - <propeller_diameter> - The diameter of the propeller in meters.
   ///   [Optional, m, defaults to 0.02m]
   /// - <thrust_coefficient> - This is the coefficient which relates the angular
-  ///   velocity to actual thrust. [Optional, no units, defaults to 1.0]
+  ///   velocity to actual thrust. A positive coefficient spins the propeller
+  ///   counter-clockwise for a positive thrust force; a negative coefficient
+  ///   spins the propeller clockwise for positive thrust force. [Optional,
```

Comment:
Just a note that "clockwise" can be confusing unless the reference frame is clear. It's worth clarifying whether this is from the point of view from the vehicle, or looking at it from the outside.

Reply:
Geez, great catch @chapulina

I meant to apply the maritime nomenclature: viewed stern to bow, a clockwise (right-hand) propeller spins clockwise to provide forward (positive) thrust. But I got it flipped. I will fix that, and note the reference frame.